plugin/ui.tsx

```diff
@@ -756,305 +850,448 @@ class App extends SafeComponent {
                       >
                         <FormattedMessage
-                          id="learnMore"
-                          defaultMessage="Learn more"
+                          id="errorMessage"
+                          defaultMessage="Oh no, there was an error! To troubleshoot, if you are
+                            importing a whole page, try importing a smaller part of the
+                            page at a time, like one section or even one button"
                         />
-                      </Button>
+                      </div>
+                      <div>
+                        <Button
+                          style={{ textTransform: "none" }}
+                          size="small"
+                          color="primary"
+                          href="https://www.builder.io/c/docs/import-from-figma#troubleshooting"
+                          target="_blank"
+                          rel="noopenner"
+                        >
+                          <FormattedMessage
+                            id="learnMore"
+                            defaultMessage="Learn more"
+                          />
+                        </Button>
+                        <Button
+                          size="small"
+                          style={{ opacity: 0.5, textTransform: "none" }}
+                          onClick={() => {
+                            this.showRequestFailedError = false;
+                          }}
+                        >
+                          <FormattedMessage
+                            id="clearErrors"
+                            defaultMessage="Clear errors"
+                          />
+                        </Button>
+                      </div>
+                    </div>
+                  )}
+
+                  {this.figmaCheckList &&
+                    Object.keys(this.figmaCheckList).length && (
+                      <div>
+                        <div
+                          style={{
+                            fontWeight: "bold",
+                            margin: "5 0 5",
+                            fontSize: 12,
+                          }}
+                        >
+                          <FormattedMessage
+                            id="contentListBeforeImport"
+                            defaultMessage="Prep your Figma file for export"
+                          />
+                        </div>
+                        {this.figmaCheckList.results?.map((item) => {
+                          if (item.data.type === "before") {
+                            return (
+                              <div
+                                key={item.id}
+                                style={{
+                                  display: "flex",
+                                  fontSize: 11,
+                                }}
+                              >
+                                <StyledCheckbox color="primary" />
```

Comment:
I was chatting with Tiana, and I think the checkboxes are going to greatly confuse the users. I'm fairly sure people will think that checking the boxes actually does something code-wise.

Tiana said she's revisiting the design (but seems to be out sick right now). I think using Bullet points will be clearer as a first step...

It's also fine to just merge this right now, and wait for her feedback before publishing the changes

Reply:
ah yes I agree with this. I imagined these boxes would get checked automatically like when your design fully uses autolayout etc. but when I heard it's powered by the CMS and you can add/remove any that means they won't have this logic so I agree I think bullets is wiser here

Reply:
Or at the very least, have them be always enabled and non-editable. They should not be interactive elements

Reply:
Changed checkboxes to bullets